### DISCUSSION TITLE
v1.16.0

### DISCUSSION THREAD

**Discussion by andygrunwald** [andygrunwald/go-jira#487] [General]
## What's Changed

* chore(deps): bump github.com/golang-jwt/jwt/v4 from 4.3.0 to 4.4.0 by @dependabot in https://github.com/andygrunwald/go-jira/pull/443
* chore(deps): bump github.com/golang-jwt/jwt/v4 from 4.4.0 to 4.4.1 by @dependabot in https://github.com/andygrunwald/go-jira/pull/445
* chore(deps): bump actions/checkout from 2 to 3 by @dependabot in https://github.com/andygrunwald/go-jira/pull/448
* chore(deps): bump actions/cache from 2 to 3.0.1 by @dependabot in https://github.com/andygrunwald/go-jira/pull/447
* GitHub Actions Update: Run tests on macOS, deprecate Go v1.15 and Go v1.16, introduce Go v1.18, run CI every night by @andygrunwald in https://github.com/andygrunwald/go-jira/pull/449
* GitHub Actions: Upgrade staticcheck action version to v1.2.0 by @andygrunwald in https://github.com/andygrunwald/go-jira/pull/451
* Added an example of how to add a label to an issue by @lucribas in https://github.com/andygrunwald/go-jira/pull/442
* GitHub Actions: Remove greetings workflow by @andygrunwald in https://github.com/andygrunwald/go-jira/pull/450
* chore(deps): bump actions/setup-go from 2 to 3 by @dependabot in https://github.com/andygrunwald/go-jira/pull/452
* chore(deps): bump github.com/google/go-cmp from 0.5.7 to 0.5.8 by @dependabot in https://github.com/andygrunwald/go-jira/pull/453
* chore(deps): bump github.com/golang-jwt/jwt/v4 from 4.4.1 to 4.4.2 by @dependabot in https://github.com/andygrunwald/go-jira/pull/464

## New Contributors
* @lucribas made their first contribution in https://github.com/andygrunwald/go-jira/pull/442

**Full Changelog**: https://github.com/andygrunwald/go-jira/compare/v1.15.1...v1.16.0

<hr /><em>This discussion was created from the release <a href='https://github.com/andygrunwald/go-jira/releases/tag/v1.16.0'>v1.16.0</a>.</em>

**Comment by cjones722** [comment]
Have question about installing go-jira on a MacBook.  What is the proper command to install?

Message "go get" is no longer supported is displayed:

$ go get github.com/andygrunwald/go-jira
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.

$ go version
go version go1.20.2 darwin/amd64

When using "go install", these messages are displayed:

$ go install github.com/andygrunwald/go-jira@latest
package github.com/andygrunwald/go-jira is not a main package

$ go install gopkg.in/andygrunwald/go-jira.v1
go: 'go install' requires a version when current directory is not in a module
	Try 'go install gopkg.in/andygrunwald/go-jira.v1@latest' to install the latest version

$ go install gopkg.in/andygrunwald/go-jira.v1@latest
go: finding module for package github.com/fatih/structs
go: finding module for package github.com/pkg/errors
go: finding module for package github.com/google/go-querystring/query
go: finding module for package github.com/trivago/tgo/tcontainer
go: found github.com/fatih/structs in github.com/fatih/structs v1.1.0
go: found github.com/google/go-querystring/query in github.com/google/go-querystring v1.1.0
go: found github.com/pkg/errors in github.com/pkg/errors v0.9.1
go: found github.com/trivago/tgo/tcontainer in github.com/trivago/tgo v1.0.7
package gopkg.in/andygrunwald/go-jira.v1 is not a main package

**Comment by vladimir-ch** [comment]
> Message "go get" is no longer supported is displayed:

The message says that 'go get' is no longer supported _outside a module_. Presumably you want to install go-jira to use it in your project, so your project should use modules and then 'go get' will work:

```
go mod init my-jira-project
go get github.com/andygrunwald/go-jira
```

> When using "go install", these messages are displayed:

The message says to use 'go install' to _build and install a command_. go-jira is a library, not an executable, so you should not use 'go install'.

**Reply by cjones722** [reply]
Thank you.
